README.md

```diff
@@ -29,6 +29,33 @@ So, plz help me (´；ω；｀)
 ## Installation
     go get -u github.com/Code-Hex/battery/cmd/battery
 
+## Support patched fonts
```

Comment:
I think better to move this section under the Usage section as `###`
And Please lightly describe in this section **how options are available** to use this font.

Reply:
OK! I will try it.